msdk-io-mgf/src/test/resources/test_query.mgf

```diff
@@ -0,0 +1,24 @@
+BEGIN IONS
```

Comment:
it is also a good idea to add some malformed MGF files as tests, and make sure your code throws the right MSDKException when executed on such file